src/app/components/modules/Flag/index.jsx

```diff
@@ -0,0 +1,8 @@
+import React, { Component } from 'react';
+
+const Flag = ({ flagged, FlagComponent, Fallback = null, children }) => {
+    if (flagged && children) return { ...children };
```

Comment:
If there are children, and there's also a FlagComponent specified it would be good for us to warn.

Reply:
i'll add to the README that child components will be used before a provided FlagComponent

Reply:
done in https://github.com/steemit/condenser/pull/2432/commits/9e4809b2bafad7805c97e665b35951e29136024c

Reply:
i'm going to add proptypes validation to the Flag component actually

Reply:
added in https://github.com/steemit/condenser/pull/2432/commits/4c86617bf4f6f76448424b7c67cb3e27d28123dd